project/src/app/core/lsb.ts

```diff
@@ -0,0 +1,32 @@
+import { chunk, dropRightWhile, flow, map, join } from "lodash/fp";
+import { AppImage } from "./app-image";
+
+export function highlightLsb(image: AppImage): AppImage {
+  return image.mapBytes((byte) => ((byte & 1) === 1 ? 255 : 0));
+}
+
+export function embedText(image: AppImage, text: string): AppImage {
+  const bits = text
+    .split("")
+    .flatMap((char) =>
+      char.charCodeAt(0).toString(2).slice(-7).padStart(7, "0").split("")
```

Comment:
Czy to nie sprawia że ograniczamy się do 128 różnych znaków?

Reply:
Tak, tylko ASCII. Można to lepiej przemyśleć.

Reply:
Jeśli rozszerzymy na całe 8 bitów, to będzie można chować w obrazach emoji albo nawet całe pliki. Jeśli mamy jakiegoś backloga, to to trzeba tam ciepnąć.